addons/pick-colors-from-stage/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Pick colors on stage with eyedropper",
+  "name": "Costume editor stage eyedropper",
```

Comment:
I don't know about this one...

Reply:
I don't like it because the addon doesn't add a "stage eyedropper" - it changes the existing eyedropper.

Reply:
```suggestion
  "name": "Pick stage colors with eyedropper",
```